.storybook/grid/GridPanel.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { AddonPanel } from '@storybook/components';
+import { useChannel } from '@storybook/api';
+import { styles } from './gridStyles';
+import { INITIAL_IMAGE_EVENT, IMAGE_SELECTED_EVENT } from './withGrid';
+import { STORY_CHANGED } from '@storybook/core-events';
+import { IframePostMessageService } from '@guardian/grid-client';
+import { Reporter } from '@guardian/grid-client/lib/utils';
+
+const GRID_URL =
+    process.env.NODE_ENV === 'development'
+        ? 'https://media.test.dev-gutools.co.uk'
+        : 'https://media.gutools.co.uk';
+
+const isValidMessage = (data) => data?.crop?.data && data?.image?.data;
```

Comment:
Is this used?

Reply:
Ah great spot, thanks, I'll remove. No it's not used any longer, `@guardian/grid-client` is now doing this for us. (I'm surprised the editor/build steps didn't shout about this, perhaps there's a linting/typescript config value we're missing to make sure it does)